core/src/main/java/com/google/zxing/pdf417/detector/PDF417DetectorResult.java

```diff
@@ -28,10 +28,12 @@ public final class PDF417DetectorResult {
 
   private final BitMatrix bits;
   private final List<ResultPoint[]> points;
+  private final int rotation;
 
-  public PDF417DetectorResult(BitMatrix bits, List<ResultPoint[]> points) {
+  public PDF417DetectorResult(BitMatrix bits, List<ResultPoint[]> points, int rotation) {
```

Comment:
`com.google.zxing.pdf417.detector.PDF417DetectorResult: In method 'public PDF417DetectorResult(com.google.zxing.common.BitMatrix, java.util.List)' the number of arguments has changed` Ah yeah it has a point. Let's keep the original constructor as well and have it default to rotation=0

Reply:
Oh yeah, great point!